scheduler/task_prioritizer.go

```diff
@@ -11,14 +11,18 @@ import (
 	"github.com/pkg/errors"
 )
 
+// this controls (I think) how the system interleaves patches and
+// commits, but it doesn't really make sense. It used to be
+// configurable, but that complicates interface complexity.
+const schedulerMergeToggle = 2
```

Comment:
should remove this from the config struct too

Reply:
I opened a separate ticket for removing this.